public-sector-websites.csv

```diff
@@ -133,6 +133,7 @@ Department of Internal Affairs,gazette.govt.nz,No,Public Service department
 Department of Internal Affairs,goodpracticefunding.govt.nz,No,Public Service department
 Department of Internal Affairs,goodpracticeparticipate.govt.nz,No,Public Service department
 Department of Internal Affairs,govtechtalent.co.nz,Yes,Public Service department
+Department of Internal Affairs,govtechtalent.govt.nz,Yes,Public Service department
```

Comment:
@racheljdavis looking at this the domain here doesn't redirect - the site serves up under the govt.nz domain. Can you double check and update to "No" if that is correct.

Reply:
Someone needs to check what they copy and paste...